doc/api.md

```diff
@@ -180,6 +180,10 @@ Some transformation plugins also allow the use of the following optional fields:
 -   **expression**: indicates that the value of the target attribute will not be the plain value or the measurement, but
     an expression based on a combination of the reported values. See the
     [Expression Language definition](#expression-language-support) for details
+-   **skipValue**: indicates the result of applying `expression` to a measure is equal to the value of `skipValue` then
+    the attribute corresponding to the measure is not sent to CB. By default if `skipValue` is not defined then is
+    considered as `null` (i.e. if the result of apply `expression` results in `null` then corresponding attribute is not
+    sent to CB).
```

Comment:
```suggestion
-   **skipValue**: indicates that if the result of applying `expression` to a measure is equal to the value of `skipValue` then
    the attribute corresponding to the measure is not sent to CB. By default if `skipValue` is not defined then is
    considered as `null` (i.e. if the result of apply `expression` results in `null` then corresponding attribute is not
    sent to CB). It is only used if `expression` is provided (otherwise is ignored).
```